bundle/src/main/java/com/adobe/acs/commons/workflow/synthetic/impl/SyntheticWorkflowModelImpl.java

```diff
@@ -33,11 +33,18 @@
 import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
+import javax.jcr.RepositoryException;
 
 public class SyntheticWorkflowModelImpl implements SyntheticWorkflowModel {
     private static final Logger log = LoggerFactory.getLogger(SyntheticWorkflowModelImpl.class);
 
-    private static final String WORKFLOW_MODEL_PATH_PREFIX = "/etc/workflow/models/";
+    private static final String[] WORKFLOW_MODEL_PATH_PREFIXES = new String[]{
+        "",
```

Comment:
This feels a bit random ...  which models do we need? 

In 6.4 - If they aren't in `/var/workflow/models` can they even be run? (Wouldnt that mean they aren't synch'd and there is nothing to operate against?)

If thats not the case, and an unsynched model is sufficient, we probably should first check:

1. /conf/global/settings/workflow/models
2. /apps/settings/workflow/models
3. /libs/settings/workflow/models

to follow CA Config look-up.

And then finally fallback to /etc/workflow/models (which per my first point on published models seems weird, since IIRC < 6.4 also synch'd them out somewhere).